effekt/shared/src/main/scala/effekt/core/vm/VM.scala

```diff
@@ -113,7 +113,7 @@ enum VMError extends Throwable {
     case VMError.MissingBuiltin(name) => s"Missing builtin: ${name}"
     case VMError.RuntimeTypeError(msg) => s"Runtime type error: ${msg}"
     case VMError.NonExhaustive(missingCase) => s"Non exhaustive: ${missingCase}"
-    case VMError.Hole() => s"Reached hole"
+    case VMError.Hole(span) => s"Reached hole @ ${span.range.from.format}"
```

Comment:
I use `span.range.from.format`, but it's a bit of a mouthful + it has a trailing `:`.
Is there a better way to get `${file}:${line}:${column}` with the new spans? cc @timsueberkrueb